### DISCUSSION TITLE
Continue numbered list with correct number after notice

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7119] [Ideas]
When editing a document and using a numbered list, you can insert a notice.  However, numbering does not correctly continue after the notice.  As far as I can see, there is no way around this.
Example:
![image](https://github.com/outline/outline/assets/25268183/d5e6500c-9b9b-4a00-adca-3446f909c601)

**Comment by tommoor** [comment]
You can type `14. ` at the beginning of the line to continue the list, this also works to start a list at any numbered point.

**Reply by designatedsuccessor** [reply]
Got it.  Thanks.  It is a little tricky, though.  We had to play with it to make it work.  At first, I literally just typed **14.**  That didn't work (presumably because the **1.** was already assumed), so we backspaced, which made the step part of the blue "note" area above it.  Then, we pressed the Enter key, and it went down a line within the blue "note."  We did eventually figure it out, but I'm still not 100% sure how we made it work, and I don't think I could reliably reproduce it again.

**Reply by tommoor** [reply]
List editing can be a little tricksy still – just keep hitting `Return` until you're back at the "root" of the document (an empty paragraph) and then type the number shortcut to start a new list

**Reply by designatedsuccessor** [reply]
I just discovered something that is a little frustrating at times: if you have several lists interrupted by, say, a info notice, the lists are "disconnected" numerically.

Example:
1. abc
2. def
3. ghi
Info: hello
4. jkl
5. mno

If you add a step between Step 2 and Step 3 (effectively creating 4 total steps before the info notice), you'll have to manually update the existing Step 4 after the info notice to Step 5.

This isn't a big deal in the example above, but it can be cumbersome if you have a quite a few notices through your long step-by-step list.

**Comment by tommoor** [comment]
This will happen if you exit the list but you can stay inside the list and
add a notice – in which case the numbering will not be reset. To achieve
this you can use a soft break – shift+enter to create a newline.
Alternative you can create the notice outside of the list and drag it in if
that proves easier.
